### PR TITLE
beta config for EJB timer fail over

### PR DESCRIPTION
**Pull request by njr-11**
Add metatype and translatable names/descriptions for the property that will gate the beta capability for EJB timer fail over.  Also, updates a couple of other related properties, maintaining compatibility with the existing single-server mode.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent.persistent/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -40,14 +40,17 @@ initialPollDelay.desc=Duration of time to wait before this instance might poll t
 jndiName=JNDI name
 jndiName.desc=JNDI name.
 
+missedTaskThreshold=Missed task threshold for fail over
+missedTaskThreshold.desc=Amount of time beyond a task execution's expected start to reserve for running it. Other members are prevented from running the task prior to the expiration of this interval. If the interval elapses without successful execution of the task, then the task execution is considered to have been missed, enabling another member to attempt to run it. This enables fail over.
```
[the remark this answers is not in the document]
Comment (njr-11) [reply dev/com.ibm.ws.concurrent.persistent/resources/OSGI-INF/l10n/metatype.properties]: Thanks for the improvements, I've made all of them as you suggested under https://github.com/OpenLiberty/open-liberty/pull/10284/commits/eeb252936b28153aa9e7cbde93e50dd03d7b2f3a

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by njr-11** [comment]
On the rebuild, all of the tests which ran were successful, but an infrastructure issue caused one of the async test builds to not run.  I have resubmitted it.

**Comment by skasund** [comment]
L2 message review not required

**Comment by njr-11** [comment]
The resubmit of the async test run was successful with all tests passing.

**Pull request merged**